src/views/Home/index.view.tsx

```diff
@@ -51,20 +21,10 @@ const HomepageView: React.FC = () => {
     <>
       <MLHBanner />
       <div className="Homepage">
-        <HeroView pageName={"Home"} title={title} description={description}>
-          <div className="Homepage__emailSubscriptionContainer">
-            <EmailSubscriptionForm />
-          </div>
-          <div className="Homepage__buttonContainer">
-            {Object.entries(buttonProps).map(([_, buttonProp]) => (
-              <Button
-                key={buttonProp.text}
-                className="Homepage__button"
-                {...buttonProp}
-              />
-            ))}
-          </div>
-        </HeroView>
```

Comment:
Not entirely sure why we got rid of this -- this PR/branch deals with the `Mission` component only, so let's put this back!

Reply:
Ahh, another Freudian slip! I was thinking that this section was going to be replaced instead of reprovisioned. Nice catch!